src/control/lib/daos/pool_property.go

```diff
@@ -27,6 +28,42 @@ func numericMarshaler(v *PoolPropertyValue) ([]byte, error) {
 	return json.Marshal(n)
 }
 
+const (
+	defDurationUnit = time.Second
+)
+
+// parseDuration parses a duration string into a time.Duration.
+func parseDuration(s string) (time.Duration, error) {
+	if s == "" {
+		return 0, nil
+	}
+
+	// Append the duration unit if it is missing.
+	if s[len(s)-1] >= '0' && s[len(s)-1] <= '9' {
+		dduStr := defDurationUnit.String()
+		s += dduStr[len(dduStr)-1:]
+	}
+
+	// For convenience, parse custom duration units beyond
+	// the standard set defined by time.ParseDuration().
+	unit := s[len(s)-1]
```

Comment:
Same problem here...

Reply:
Well, in this case, we're not overriding `time.ParseDuration()`, we're just looking for a trailing 'd' or 'w' to make it easier to specify longer scrubber intervals. If we don't see either of those single-letter custom units, then we fall back to the standard parsing behavior.

Reply:
Ah, I see. Thanks for clarifying.